simple_template_server/README.md

```diff
@@ -0,0 +1,50 @@
+Template Development Server
+===================================
+
+This is a simple Snippet Server for development of Snippet Templates. It ignores
+the client configuration including locale, Firefox version and release channels
+and just returns all the snippets to the requesting client. If you only care for
+Snippet Template development this is the server for you.
+
+Installation
+------------
+
+1. Create and activate a Python virtualenv
+
+    ``` shell
+    virtualenv venv
+    . venv/bin/activate
+    ```
+
+2. Install the needed packages
+
+    ``` shell
+    pip install -r requirements.txt
+    ```
+
+Using the server
+----------------
+
+1. Run the server
+
+    ``` shell
+    python webserver.py
+    ```
+
+    The server listens on *http://0.0.0.0:8000*
```

Comment:
When I navigate to http://0.0.0.0:8000/, I get a 500.

My console is reporting:

```
(venv)[scott@localhost snippets-service]$ python ./simple_template_server/webserver.py 
Bottle v0.12.7 server starting up (using WSGIRefServer())...
Listening on http://0.0.0.0:8000/
Hit Ctrl-C to quit.

Traceback (most recent call last):
  File "/home/scott/snippets-service/venv/lib/python2.7/site-packages/bottle.py", line 862, in _handle
    return route.call(**args)
  File "/home/scott/snippets-service/venv/lib/python2.7/site-packages/bottle.py", line 1729, in wrapper
    rv = callback(*a, **ka)
  File "./simple_template_server/webserver.py", line 92, in home
    template = jinja_env.get_or_select_template('fetch_snippets.jinja')
  File "/home/scott/snippets-service/venv/lib/python2.7/site-packages/jinja2/environment.py", line 830, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/home/scott/snippets-service/venv/lib/python2.7/site-packages/jinja2/environment.py", line 791, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/home/scott/snippets-service/venv/lib/python2.7/site-packages/jinja2/environment.py", line 765, in _load_template
    template = self.loader.load(self, name, globals)
  File "/home/scott/snippets-service/venv/lib/python2.7/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/home/scott/snippets-service/venv/lib/python2.7/site-packages/jinja2/loaders.py", line 178, in get_source
    raise TemplateNotFound(template)
TemplateNotFound: fetch_snippets.jinja
127.0.0.1 - - [11/May/2016 09:15:11] "GET / HTTP/1.1" 500 741

```